apps/src/templates/SearchBar.jsx

```diff
@@ -31,32 +31,33 @@ const styles = {
     top: 9,
     left: 10
   },
-  search: {
+  searchArea: {
     position: 'relative'
   }
 };
 
 export default class AnimationPickerSearchBar extends React.Component {
   static propTypes = {
-    value: PropTypes.string.isRequired,
+    placeholderText: PropTypes.string.isRequired,
+    styles: PropTypes.object.isRequired,
```

Comment:
This feels a little weird but I can't 100% put my finger on why. Allowing the consumer to style different elements in the component makes sense, and so does combining all those properties into one object (I think). Maybe we should be explicit about what keys it takes:

```suggestion
    styles: PropTypes.object.shape({
      searchArea: PropTypes.object,
      icon: PropTypes.object,
      input: PropTypes.object
    });
```

What do you think?

Reply:
oo, didn't know you could specify what keys the object has! Yeah that definitely feels better, thanks!

Reply:
Do all of these styles _need_ to be customized? Or can we make smart defaults, and have them be the same everywhere `<SearchBar/>` gets used?